assembl/static2/css/brightMirror.scss

```diff
@@ -26,20 +26,56 @@ $fiction-toolbar-icon-size: 17px;
   }
 }
 
+:focus {
+  outline-color: $white;
```

Comment:
`outline: none` is better no? Because if you change the background-color of the page, the outline will be visible